components/webhooks/cmd/serve.go

```diff
@@ -1,30 +1,68 @@
 package cmd
 
 import (
+	"strings"
 	"syscall"
+	"time"
 
 	"github.com/formancehq/stack/libs/go-libs/logging"
 	"github.com/formancehq/stack/libs/go-libs/service"
 	"github.com/formancehq/webhooks/cmd/flag"
 	"github.com/formancehq/webhooks/pkg/otlp"
 	"github.com/formancehq/webhooks/pkg/server"
+	"github.com/formancehq/webhooks/pkg/storage/postgres"
+	"github.com/formancehq/webhooks/pkg/worker"
+	"github.com/pkg/errors"
 	"github.com/spf13/cobra"
 	"github.com/spf13/viper"
+	"go.uber.org/fx"
 )
 
-var serverCmd = &cobra.Command{
-	Use:   "serve",
-	Short: "Run webhooks server",
-	RunE:  RunServer,
+func newServeCommand() *cobra.Command {
+	return &cobra.Command{
+		Use:   "serve",
+		Short: "Run webhooks server",
+		RunE:  serve,
+	}
 }
 
-func RunServer(cmd *cobra.Command, _ []string) error {
+func serve(cmd *cobra.Command, _ []string) error {
 	logging.FromContext(cmd.Context()).Debugf(
-		"starting webhooks server module: env variables: %+v viper keys: %+v",
-		syscall.Environ(), viper.AllKeys())
+		"starting webhooks server module: env variables: %+v viper settings: %+v",
+		syscall.Environ(), viper.AllSettings())
 
-	return service.New(cmd.OutOrStdout(),
+	retriesSchedule := make([]time.Duration, 0)
+	rs := viper.GetString(flag.RetriesSchedule)
```

Comment:
same comment as gfyrag here, maybe we can use string slice or duration slice from viper !